modules/builds-using-image-change-triggers.adoc

```diff
@@ -28,7 +28,7 @@ metadata:
 +
 This defines the image stream that is tied to a container image repository located at `<system-registry>_/_<namespace>_/ruby-20-centos7`. The `<system-registry>` is defined as a service with the name `docker-registry` running in {product-title}.
 
-. If an image stream is the base image for the build, set the from field in the build strategy to point to the `ImageStream`:
+. If an image stream is the base image for the build, set the `from` field in the build strategy to point to the `ImageStream`:
```

Comment:
Up at line 29, not sure why namespace is in italics but system-registry is not?  It looks like both are values the user would replace.

Reply:
Updated.